ci/run_envoy_docker.sh

```diff
@@ -22,4 +22,5 @@ fi
 mkdir -p "${ENVOY_DOCKER_BUILD_DIR}"
 # Since we specify an explicit hash, docker-run will pull from the remote repo if missing.
 docker run --rm -t -i -u "${USER}":"${USER_GROUP}" -v "${ENVOY_DOCKER_BUILD_DIR}":/build \
-  -v "$PWD":/source -e NUM_CPUS "${IMAGE_NAME}":"${IMAGE_ID}" /bin/bash -lc "cd source && $*"
+  -v "$PWD":/source -e NUM_CPUS --cap-add SYS_PTRACE "${IMAGE_NAME}":"${IMAGE_ID}" \
```

Comment:
Do you happen to know if this will cause any issues on OSX? Should we do it only on Linux?

Reply:
Sorry, I don't, my Docker setup on OSX is busted. Maybe @zuercher knows and/or can verify?

Reply:
According to https://github.com/docker/for-mac/issues/1564 it looks like it won't break the CLI (but might not work) so OK by me to merge.